debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -159,13 +161,30 @@ public void commit() throws InterruptedException {
         }
     }
 
+    @Override
+    public void commitRecord(SourceRecord record) throws InterruptedException {
+        if (running.get()) {
```

Comment:
I don't think you need to bother with this check; there's no side effect where it matters, commit() will deal with it.

Reply:
Agreed :+1: